static/js/server_events_dispatch.js

```diff
@@ -329,6 +329,8 @@ export function dispatch_normal_event(event) {
                     bot_data.add(event.bot);
                     settings_bots.render_bots();
                     settings_users.update_bot_data(event.bot.user_id);
+                    // Function redraw_bots_list() is not loading latest added bot.
+                    settings_users.load_bots_table();
```

Comment:
We should debug why `settings_users.update_bot_data` isn't working, not adding a duplicate rerendering. As noted above, the `render_item` function should add the new bot.

Reply:
OK I figured out what the actual bug was here, and fixed it in 09c6ee6468e302b5c9d09fb40e6955c1d928e408